layersvt/device_simulation.md

```diff
@@ -364,3 +271,7 @@ The DevSim layer itself does very little sanity-checking of the configuration fi
 ### Other Resources
 1. http://json.org/
 2. http://json-schema.org/
+
+## Layer Options
+
+The options for this layer are specified in VK_LAYER_LUNARG_device_simulation.json. The option details are in [device_simulation_layer.html](https://vulkan.lunarg.com/doc/sdk/latest/windows/device_simulation_layer.html#user-content-layer-details).
```

Comment:
Change "The option details are in [device_simulation_layer.html] to ...

"The layer option details can be found in the [device simulation layer documentation]"